dev-docs/bidders/freewheelssp.md

```diff
@@ -21,5 +21,47 @@ sidebarType: 1
 
 {: .table .table-bordered .table-striped }
 | Name     | Scope    | Description | Example | Type     |
-|----------|----------|-------------|---------|----------|
-| `zoneId` | required |             |         | `string` |
+|----------|----------|-------------------------|--------|----------|
+| `zoneId` | required | The zone ID for the ad. | "2003" | `string` |
+| `format` | optional | The format to use for displaying the ad. Can be one of the following:<br/>* screen-roll<br/>* intext-roll<br/>* sliderad<br/>* floorad<br/>* expand-banner<br/>* instream<br/>* inbanner<br/>Note: The screen-roll, intext-roll, sliderad and floorad formats are all FreeWheel outstream formats.<br>Default value: "instream" | "screen-roll" | `string` |
+| `bidfloor` | optional | Bid floor price. | 13.2118  | `float` |
```

Comment:
Note that accepting bidfloor as a bidder-specific parameter is ok as long as the adapter can also read it from the floors module.

Reply:
Hi Bretg, thanks for the notice. We've supported the dynamic floor price in our adapter. Do you suggest to keep the 'bidfloor' param here or remove it?

Reply:
As long as you support the standard way, a bidder-specific way is ok.